mikeio1d/res1d.py

```diff
@@ -157,25 +156,11 @@ def warn_deprecation(name: str, hint: str = ""):
             time=time,
         )
 
-        self.result_network = ResultNetwork(self)
-        self.network = self.result_network
+        self._network = ResultNetwork(self)
```

Comment:
Do we really want to underscore this one? It is not private to Res1D class, but internal to mikeio1d.

Reply:
Good point. I put it as a read only property instead. But I guess it's also fine to leave it as something that someone could hack with if they want. Subclassing and creating their own super network object 😆